### ISSUE TITLE
Productive Bee Sorting Delete Items

### ISSUE THREAD

**Issue by Medivha** [FTBTeam/FTB-Modpack-Issues#4095]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.2.0

### Describe the bug

When you sort (middle mouse button) in the Advanced Beehive GUI all combs and upgrades get voided/deleted/not accessible

bee's stay

idk which of the sorting mods does

### Steps to reproduce

have produced good/upgrades inyour advanced beehive
press the sorting button
swooosh

### Expected behaviour

not deleting stuff ;)

### Additional information

the Author of Productive Bee's mentioned the modpack teams have to black list the GUI

[https://github.com/JDKDigital/productive-bees/issues/406](url)

**Comment by DinnerBeef** [comment]
Sorry about that. I forget to change something, so the config is pushed. Should be fixed in the next pack update.

**Comment by Medivha** [comment]
the bug is still present in 1.3.1 (i patched from 1.20 to 1.31 because you know ;) so could not test in 1.3.0 )

tested with 4 upgrades in the right slot (simulator and 3 productivity alpha) and 2 stacks of produced Comb's in the outcome slots
pressed the sorting middle mouse and all is gone

i was testing it before with just one upgrade on the right and nothing in the outcome slot and it moved into the bee cage slot and was thinking first OK weird but seemed fixed. But now out of habit (with pressing sorting in inventorys) with the above scenario (4 upgrade 2 outcome) it got voided again

edit: i did another Test in a new created world with creative, and it works no deleting.

 so how do i apply it to my save?!

another edit: i found the config and compared it to my current world and the new created. So it doesn't rewrite the config with the patch. 
![](https://i.imgur.com/5Oj0fo1.png)

i added "productivebees:advanced_beehive" myself and now it works.
idk if you can force the config change to other people. (but kinda also makes sense that normally it doesn't, so it does not mess with people's settings)

**Comment by Medivha** [comment]
As stated above, I added it myself and fixed it with it. It was more of a concern left for old versions for other people. Because your fix did not update at that time, the config in saves. You added it to the default config(I assume) in the pack, but it was not forcing it to the config in saves.

To test this, I would need to create an old version of a pack create a world so it has the old config and then update to see if it overrides in the save.
 I don't have the time for that right now, and I stopped playing it. It would only if it is an issue effect old worlds that are updated and not created after this update.
